### ISSUE TITLE
leveldb: prepend user dir to relative db path?

### ISSUE THREAD

**Issue by boneskull** [node-red/node-red-nodes#403]
When the leveldb node is given a relative path, e.g. `my-dir/my-db`, perhaps it should prepend `RED.settings.userDir` to this path.  This is the behavior that I'd expect, anyhow.

Otherwise, leveldown *appears* to want to use the filesystem root, and will throw an exception if NR is not running as superuser.  If NR *is* running privileged, then the behavior is likely undesirable anyhow.

This should not affect absolute paths such as `/home/me/.node-red/my-db`.  That's a little tedious to type, as well.  :wink:

I can send a PR if this would be desirable--but it *is* a breaking change.

(Alternatively, maybe just add some docs to the info pane.)

**Comment by dceejay** [comment]
I agree relative makes more sense - but need to check what other nodes already do... eg file node etc...
So something we probably need to consider more widely to be honest... 
I suspect for now better docs is the right answer for immediate PR.
Then follow up with more changes. We can of course do a major version bump if we need to.
